docs/sources/send-data/otel/_index.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Ingesting logs to Loki using OpenTelemetry Collector
+menuTitle:  Ingesting OpenTelemetry logs to Loki
+description: Ingesting logs to Loki using OpenTelemetry Collector
+aliases: 
+- ../clients/k6/
+weight:  250
+---
+
+# Ingesting logs to Loki using OpenTelemetry Collector
+
+{{% admonition type="warning" %}}
+OpenTelemetry logs ingestions is an experimental feature and is subject to change in future releases of Grafana Loki.
+{{% /admonition %}}
+
+Loki now natively supports ingesting OpenTelemetry logs over HTTP.
+For ingesting logs to Loki using OpenTelemetry Collector, you need to use the [`otlphttp`](https://github.com/open-telemetry/opentelemetry-collector/tree/main/exporter/otlphttpexporter) exporter.
+
+## Loki configuration
+
+When logs are ingested to Loki using OTLP ingestion endpoint, some of the data is stored as [Structured Metadata]({{< relref "../../get-started/labels/structured-metadata" >}}).
+Since Structured Metadata is still an experimental feature, Loki by defaults rejects any writes using that feature.
+To start ingesting logs in OpenTelemetry format, you need to enable `allow_structured_metadata` per tenant configuration (in the `limits_config`).
+
+## Configure the OpenTelemetry Collector to write logs into Loki
+
```

Comment:
This needs an introductory sentence or two to explain that the following edits are made in the collector config file.  It probably wouldn't hurt to drop in a link to https://opentelemetry.io/docs/collector/configuration/.